example/server/integrations.py

```diff
@@ -970,3 +988,161 @@ def info_integration(request: Request, asset: Asset, lang: str):
+            # This method is not supported
+            return False
+    return True
+
+
+def check_country_code(asset: str, country_code: str) -> bool:
+    if country_code and country_code not in get_offchain_asset(
+        asset
+    ).country_codes.split(","):
+        # The contry code is not supported by the anchor.
+        return False
+    return True
+
+
+class MySEP38AnchorIntegration(SEP38AnchorIntegration):
+    def get_prices(
+        self,
+        sell_asset: str,
+        sell_amount: str,
+        sell_delivery_method: str = None,
+        buy_delivery_method: str = None,
+        country_code: str = None,
+    ) -> List[GetPricesResponse]:
+        exchange_pairs = list_exchange_pairs(buy_asset=sell_asset)
+
+        prices = []
+        for exchange_pair in exchange_pairs:
+            if is_stellar_asset(sell_asset):
+                # We are selling a stellar asset. Buying asset is offchain.
+                if not check_delivery_method(
+                    asset=exchange_pair.sell_asset,
+                    list_delivery_methods_func=get_buy_delivery_methods,
+                    delivery_method=buy_delivery_method,
+                ):
+                    continue
+
+                if not check_country_code(
+                    asset=exchange_pair.sell_asset, country_code=country_code
+                ):
+                    continue
+            else:
+                # We are selling an offchain asset.
+                if not check_delivery_method(
+                    asset=exchange_pair.buy_asset,
+                    list_delivery_methods_func=get_sell_delivery_methods,
+                    delivery_method=sell_delivery_method,
+                ):
+                    continue
+                if not check_country_code(
+                    asset=exchange_pair.buy_asset, country_code=country_code
+                ):
+                    continue
+
+            indicative_price = GetPricesResponse()
+            indicative_price.asset = exchange_pair.sell_asset
+            price = get_mock_indicative_exchange_price()
+            indicative_price.decimals = get_significant_decimals(indicative_price.asset)
+            indicative_price.price = to_decimals(
+                float(price), indicative_price.decimals
+            )
+            prices.append(indicative_price)
+        return prices
+
+    def get_price(
+        self,
+        sell_asset: str,
+        buy_asset: str,
+        sell_amount: str = None,
+        buy_amount: str = None,
+        sell_delivery_method: str = None,
+        buy_delivery_method: str = None,
+        country_code: str = None,
+    ) -> GetPriceResponse:
+        exchange_pairs = list_exchange_pairs(buy_asset=sell_asset, sell_asset=buy_asset)
+        # Must have exactly only one price found.
+        if len(exchange_pairs) == 0:
+            raise ValidationError(f"No price is found:  {sell_asset}->{buy_asset}")
```

Comment:
This is another new pattern; raising Django's `ValidationError` exception vs. a builtin exception like `ValueError`. Also, can't Polaris validate this before calling `get_price()`? If so, then we can validate it in the endpoint code and mention in the docstring that this validation has taken place already.